### DISCUSSION TITLE
How do i detect liveness

### DISCUSSION THREAD

**Discussion by rehkmansa** [vladmandic/human#412] [Q&A]
I've recently run into this issue for a while in which i'm trying to detect liveness on the web, I've had to port from faceapi.js to your own and now to human. Is there any way i can detect liveness. I see the demos have a roll, angle and pitch, is there any possible way i can get this values and use them to detect liveness

**Comment by vladmandic** [comment]
i suggest to take a look at `demo/faceid`, it implements several methods combined to perform validation and liveness checks.
